std/string.d

```diff
@@ -3641,25 +3671,13 @@ in
 }
 body
 {
-    import std.exception : assumeUnique;
-
-    char[] transTable = new char[256];
+    immutable(char)[256] result = void;
```

Comment:
This could use a comment saying that the array is declared as unique rather than casting the return value to make NRVO work.